ObjcVoiceQuickstart/ViewController.m

```diff
@@ -232,15 +234,37 @@ - (void)credentialsUpdated:(PKPushCredentials *)credentials {
              } else {
                  NSLog(@"Successfully registered for VoIP push notifications.");
                  
-                 /*
-                  * Save the device token after successfully registered.
-                  */
+                 // Save the device token after successfully registered.
                  [[NSUserDefaults standardUserDefaults] setObject:cachedDeviceToken forKey:kCachedDeviceToken];
+
+                 // Save the date and time to make sure we register before binding enxpiry of 365 days
```

Comment:
The date and time should be updated whenever a VoIP push arrives. That resets the TTL by 1 year. You should also update documentation to reflect that in other parts of this PR.